umbraco-cms/tutorials/editors-manual/working-with-content/rich-text-editor.md

```diff
@@ -30,211 +30,229 @@ To make your work easier, there are shortcut keys for certain editor functions.
+
+### Remove Formatting
+
+![removeFormat.jpg](images/removeFormat.jpg)
+
+If you have formatted a paragraph or selection using the formatting buttons, you can remove a formatting rule:
 
-If you are proficient in the use of HTML you can switch to HTML mode and create your page by writing HTML code. You can also check the code and make minor adjustments here to get the page exactly as you want it. Certain elements such as scripts will not be allowed and will be filtered out of code pasted into the rich text editor.
+1. Select the text and click the relevant formatting button to remove that formatting rule.
+2. You can also use the **Remove Format** button from the toolbar. If you do not have this button please speak to your system administrator.
 
 ## Links
 
 ![linkButton.jpg](images/linkButton.jpg)
 
-The _**Insert Link**_ button is used to create links to internal and external pages and media files email links and anchors. The process for inserting a hyperlink differs depending on the type of hyperlink you wish to create.
+The **Insert/Edit Link** button is used to create links to internal pages, external pages, media files, email links, and anchors. The process for inserting a hyperlink differs depending on the type of hyperlink you wish to create:
 
-### Link to a Page on Another Website
+### Link to a Page on another Website
 
 1. Select the text that will form the hyperlink.
-2. Click the _**Insert Link**_ button which will reveal the link properties slide out menu.
-3. In the URL field, enter the URL of the web page you wish to link to.
-4. In the Title field, enter the text that will be shown as a pointer to the link. This is important information for everyone reading the website with different accessibility aids.
-5. In the Target field, select the target window/frame then select how the should be opened. There are three different ways the link can be opened:
-   * Open link in a new tab in the same browser window.
-   * Open link in a new separate browser window.
-   * Open the link in the same browser tab as the user would be viewing your site.
-6. Click _**Select.**_ Your text will now contain the link as requested.
+2. Click the **Insert/Edit Link** button to open the link properties slide-out menu.
+3. In the **Link** field, enter the URL of the web page you wish to link to.
+4. In the **Link Title** field, enter the text that will be shown as a pointer to the link. This is important information for everyone reading the website with different accessibility aids.
+5. In the **Target** field, select the target window/frame to open the link in a new window or tab.
```

Comment:
Start each bullet with the "action" word 💪 
Also, add the second sentence as a sub-item - just to keep the list more concise!